gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/alttext/AltTextUiState.kt

```diff
@@ -2,7 +2,11 @@ package com.gravatar.quickeditor.ui.alttext
 
 internal data class AltTextUiState(
     val avatarUrl: String,
-    val isSaveButtonEnabled: Boolean,
     val isUpdating: Boolean,
     val altText: String,
-)
+    val altTextMaxLength: Int,
+    private val initialAltText: String = altText,
+) {
+    val isSaveButtonEnabled: Boolean
+        get() = altText != initialAltText && !isUpdating
```

Comment:
We don't need to set this manually, so I moved it here and set it conditionally.